README.md

```diff
@@ -36,10 +36,19 @@ details.
 
 Although this project is not part of the [CNCF][cncf], we abide by its [code of conduct][cncf-conduct].
 
+## License
+
+This software is licensed under the [Apache License, Version 2.0][asl2].
+
+Additionally, the End User License Agreement included in the [`EULA.pdf`](./EULA.pdf) file applies to software artifacts
+released by TriggerMesh Inc.: compiled executables, container images, etc.
```

Comment:
```suggestion
released by TriggerMesh Inc.: compiled executables and container images.
```

Reply:
Applied in [ebe1f00..6d0ff0f2](https://github.com/triggermesh/triggermesh/compare/ebe1f00..6d0ff0f2)